packages/block-library/src/post-featured-image/index.php

```diff
@@ -49,22 +49,27 @@ function render_block_core_post_featured_image( $attributes, $content, $block )
 	if ( $is_link ) {
 		$link_target    = $attributes['linkTarget'];
 		$rel            = ! empty( $attributes['rel'] ) ? 'rel="' . esc_attr( $attributes['rel'] ) . '"' : '';
+		$height         = ! empty( $attributes['height'] ) ? 'style="' . esc_attr( safecss_filter_attr( 'height:' . esc_attr( $attributes['height'] ) ) ) . '"' : '';
```

Comment:
```suggestion
		$height         = ! empty( $attributes['height'] ) ? 'style="' . esc_attr( safecss_filter_attr( 'height:' . $attributes['height'] ) ) . '"' : '';
```

I don't think we need the inner `esc_attr` calls here.

Reply:
I copied this from the suggestions from @jorgefilipecosta.  Looking at the other places in the plugin where 
`safecss_filter_attr` is used, I am seeing both escaped and unescaped attributes values, so which is correct?

Reply:
> I copied this from the suggestions

Understood. I think there was a small copy and paste error within that suggestion.

There are a few inconsistencies within the Gutenberg codebase around when we escape using `esc_attr` combined with `safecss_filter_attr`. Whether you escape the attribute before `safecss_filter_attr` or after, you'd only need one.

I'd recommend using `esc_attr( safecss_filter_attr( ... ) )` as it follows the principle of escaping as late as possible. This was recommended to me in feedback by contributors more well versed in security best practices than I.

Reply:
Then let's go with that :)